### DISCUSSION TITLE
Gitlab CI is back

### DISCUSSION THREAD

**Discussion by 0penBrain** [FreeCAD/FreeCAD#9791] [Maintainers discussions]
@FreeCAD/maintainers does one re-enabled the Gitlab CI? It seems to be back sending mails and comments on PR.

**Comment by chennes** [comment]
Not me -- and I think we should turn it back off.

**Comment by adrianinsaval** [comment]
where was the setting for this? Perhaps what's missing is that we didn't remove stuff on the gitlab side

**Reply by 0penBrain** [reply]
@chennes previously disabled the webhook. So nothing was done in Gitlab side but it can't trigger except if someone re-enables the webhook.

**Comment by adrianinsaval** [comment]
https://github.com/FreeCAD/FreeCAD/pull/9793#issuecomment-1597605938
I'm guessing @berndhahnebach did not know we had intentionally disabled this and enabled it again when the token expired (IIRC it had to be renewed periodically).

**Comment by luzpaz** [comment]
Just slightly off-topic, but where is the code that triggers FreeCAD-CI to leave messages in the FreeCAD repo issue queue ?

**Reply by 0penBrain** [reply]
Not sure to get what you mean 😕

**Reply by adrianinsaval** [reply]
maybe on gitlab or locally on bernd's pc that runs the gitlab CI.

**Comment by adrianinsaval** [comment]
so this was fully disabled again I think so we can mark this as closed
